awx/main/models/inventory.py

```diff
@@ -1547,6 +1572,13 @@ def clean_source_path(self):
             raise ValidationError(_("Cannot set source_path if not SCM type."))
         return self.source_path
 
+    '''
+    RelatedJobsMixin
+    '''
+    def _get_active_jobs(self):
+        return InventoryUpdate.objects.filter(status__in=ACTIVE_STATES,
+                                              inventory_source=self)
```

Comment:
Just saying - we need to be careful to note / remember that this method may return a queryset of either UnifiedJobs or a subclass

After talking through it with you, I see no reason to return the full polymorphic queryset. This is sufficient for almost all things we need it for. This is a rare case where using `get_real_instance` after the fact would actually be more performant. Good to note that detail, but this design is good.